index.js

```diff
@@ -45,9 +48,15 @@ console.log('hello world')
  *
  * Example Output: "$49.90"
  */
+ var price = 5.98;
+ var quantity = 4;
+ var total = price * quantity;
+ console.log(Math.round(total));
```

Comment:
in line 54 Math.round() make a number rounded. we can use .toFixed(2) to get 2 decimal https://www.w3schools.com/JSREF/jsref_tofixed.asp